### ISSUE TITLE
Version flags (-v and --version)

### ISSUE THREAD

**Issue by iTrooz** [AppImageCrafters/appimage-builder#232]
appimager-builder currently doesn't have a version flag. I think it would be pretty useful to have one, that would display appimage-builder (and appimagetool ?) versions

I'm willing to submit a PR for this, if needed

**Comment by azubieta** [comment]
You're right, feel free to send the PR. Would it be possible to fetch it from [here](https://github.com/AppImageCrafters/appimage-builder/blob/main/setup.py#L22)?

**Comment by iTrooz** [comment]
Fetching it from setup.py doesn't seems possible
This problem seems to be common with python packages, and there are a few way to go around it : https://packaging.python.org/en/latest/guides/single-sourcing-package-version/

Personally, I think using a separate version.txt file to hold to version (that would be read in setup.py and __main__.py) would be the best approach. What do you think ?

**Comment by azubieta** [comment]
This looks good as we already use git. https://pypi.org/project/setuptools-scm/
